launch/launch/actions/execute_process.py

```diff
@@ -187,7 +188,9 @@ def __init__(
             'emulate_tty' configuration does not represent a boolean.
         :param: prefix a set of commands/arguments to preceed the cmd, used for
             things like gdb/valgrind and defaults to the LaunchConfiguration
-            called 'launch-prefix'
+            called 'launch-prefix'. Note that a non-default prefix provided in
+            a launch file will override the prefix provided via the `--launch-prefix` CLI
+            argument regardless of whether `--launch-prefix-filter` is provided.
```

Comment:
@camm73 nit: I think it may be slightly more accurate to talk about launch configurations rather than CLI arguments. `ros2launch` is not the only way one can execute launch files.

Reply:
Good point. I'll adjust the wording of this